int8 LEFT=0
int8 RIGHT=1

int8 foot_lr
geometry_msgs/Pose2D foot_pose
